lib/build/index-static.jsx

```diff
@@ -38,7 +38,7 @@ function generateWithPageHTML(url, options, pageHTML) {
   options = _.defaults(options || {}, {
     meta: {}
   });
-  var locale = url.split('/')[0];
+  var locale = options.locale || 'en-US';
```

Comment:
Not that I want to remove it, but there shouldn't be a case where we need to fall back to 'en-US' at this point, right? Or does testing need it?